components/databases.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Databases
+description: Add databases to your Architect Components
+---
+
+<Warning>
+  Declarative databases are currently an **alpha** feature. While the feature should be stable, it may
+  be changed without prior notice.
+</Warning>
+
+Databases represent a common building block in cloud architecture, and Architect makes it easier than
+ever for developers to create applications that require databases. Simply declare them as part of your
+component and they'll be automatically deployed alongside your applications.
+
+```yaml architect.yml
+databases:
+  main:
+    description: Main storage for my application
+    type: postgres:13
+```
+
+## How databases get deployed
+
+Today, Architect deploys databases by transforming the database claims into single-interface services.
```

Comment:
I would say "definition" instead of "claims" here.

This page is not linked in the nav.

Reply:
How is this different from adding a `host` property to the interface? Look at the External services page. This seems to be a similar feature. I saw the term "virtual nodes" on that page and removed it because Zach said it wasn't an officially used marketing term. Is "virtual service" a term we are using officially?

Reply:
> How is this different from adding a host property to the interface? Look at the External services page. This seems to be a similar feature. I saw the term "virtual nodes" on that page and removed it because Zach said it wasn't an officially used marketing term. Is "virtual service" a term we are using officially?

The production usage is identical to that feature, yes. The reason we introduced this databases feature separately was for two reasons:

1. to simply the way you can declare common service declarations (like databases) that are in lots of components, and
2. to give Architect first-class understanding of the type of resource you're requesting. This will later be used to offer non-k8s deployment capabilities like provisioning RDS instances instead of dockerized services. This isn't supported yet, but that feature won't impact the component spec or framework at all. We would never be able to do that without knowing the difference between a service and a database.